include/nbl/builtin/hlsl/member_test_macros.hlsl

```diff
@@ -67,5 +73,94 @@ NBL_GENERATE_MEMBER_TESTER(y)
 NBL_GENERATE_MEMBER_TESTER(z)
 NBL_GENERATE_MEMBER_TESTER(w)
 
+
+// Even though it should work for some reason tests fail
+// proof it works : https://godbolt.org/z/EzPWGnTPb
```

Comment:
whats the comment about?

Reply:
I wrote that comment when I couldn't get SFINAE stuff inside nabla (because of false_type typo retrospectively) while it was working on godbolt

Reply:
so remove I guess?